stories/assets/locale.ts

```diff
@@ -9,7 +9,7 @@ const locale: Record<string, string> = {
   activityNote: 'To protect your privacy, this Brave Rewards statement is not saved, recorded or logged anywhere other than on your device (this computer). It cannot be retrieved from Brave in the event of data loss on your device.',
   addFunds: 'add funds',
   addFundsFAQ: 'the FAQ',
-  addFundsNote: 'Reminder: The Brave Wallet is unidirectional and BAT flows to publisher sites. For more information about Brave Payments, please visit',
+  addFundsNote: 'Reminder: The Brave Wallet is unidirectional and BAT flows to Creators\' sites. For more information about Brave Rewards, please visit',
```

Comment:
"creator" is a common noun. Unless there is a design specification that says otherwise, we should lowercase it.

Reply:
this should definitely be tackled shortly on the design language initiative. could punt for now but the thinking is in the context of Brave products we want to brand them as Brave Creators instead of just content creators. i.e. they aren't just another creator they are a Creator.

Reply:
Of course. Desing language not being my arena, but in my personal opinion, I would consider using `Brave Creators` specifically and `creators` when the brand is not included

Reply:
that's safer for the interim, i'll amend